ddl/ddl_api.go

```diff
@@ -1021,7 +1021,11 @@ func checkCharsetAndCollation(cs string, co string) error {
 // handleAutoIncID handles auto_increment option in DDL. It creates a ID counter for the table and initiates the counter to a proper value.
 // For example if the option sets auto_increment to 10. The counter will be set to 9. So the next allocated ID will be 10.
 func (d *ddl) handleAutoIncID(tbInfo *model.TableInfo, schemaID int64) error {
-	alloc := autoid.NewAllocator(d.store, tbInfo.GetDBID(schemaID))
+	isAutoIncColUnsigned := false
```

Comment:
I think it is better to extract these lines to a function.

Reply:
you mean line1024~1028?

Reply:
Yes, I see some similar code in this PR.